WCFDataService/Client/System/Data/Services/Client/QueryResult.cs

```diff
@@ -253,38 +254,22 @@ internal void ExecuteQuery()
                     }
                 }
 #endif
-                IODataResponseMessage response = null;
-                response = this.RequestInfo.GetSyncronousResponse(this.Request, true);
+                IODataResponseMessage response = this.RequestInfo.GetSyncronousResponse(this.Request, true);
                 this.SetHttpWebResponse(Util.NullCheck(response, InternalError.InvalidGetResponse));
 
                 if (HttpStatusCode.NoContent != this.StatusCode)
                 {
-                    using (Stream stream = this.responseMessage.GetStream())
+                    Stream stream = this.responseMessage.GetStream();
```

Comment:
Hi Uffe, I ran a full test on this change, and unfortunately it breaks many cases. The reason is that the test code internally implements a logging stream for response and add log for every Read(). And when GetStream() from the logging stream, it will return the logged bytes. So before this change, our code will copy stream which will call Read() so every bytes will be logged. Since now there is no copy, then the GetStream() cannot return correct result. It may be a very corner scenario, but anyway it is a behavior change. So to keep the change, can we add an option to turn on null stream copy? What do you think?